src/components/ContentUploader/DroppableContent.js

```diff
@@ -17,35 +17,16 @@ const dropDefinition = {
     /**
      * Validates whether a file can be dropped or not.
      */
-    dropValidator: (props, dataTransfer) => {
-        const { allowedTypes } = props;
-        return [].some.call(dataTransfer.types, (type) => allowedTypes.indexOf(type) > -1);
-    },
+    dropValidator: ({ allowedTypes }, dataTransfer) =>
+        [].some.call(dataTransfer.types, (type) => allowedTypes.indexOf(type) > -1),
```

Comment:
Probably be consistant here, either use Array.prototype or []. style.
Here vs https://github.com/box/box-ui-elements/pull/418/files#diff-1b8cad2a7eb995da9758de8009c95af9R56

Reply:
replace all `Array.prototype.` with `[].` in elements